saleor/core/utils/random_data.py

```diff
@@ -88,6 +90,25 @@
         'images_dir': 'books/',
         'is_shipping_required': True}}
 
+DEFAULT_MENU_SCHEMA = {
```

Comment:
I think we could show how different this menu is than the previous one, by adding Collections or Pages here

Reply:
Yes, it will be added :) Now you can pass only raw URL when creating new menu item, so I limited this schema only to categories, but there will be a possibility to choose category or collection from list.